agents/ocf/HealthCPU.in

```diff
@@ -217,5 +211,3 @@ esac
 rc=$?
 ocf_log debug "${OCF_RESOURCE_INSTANCE} $__OCF_ACTION : $rc"
 exit $rc
-
-# vim: set filetype=sh expandtab tabstop=4 softtabstop=4 shiftwidth=4 textwidth=80:
```

Comment:
This one ain't bad. I can't remember if you're a vim guy or an emacs guy, but vim reads this comment and applies the settings.

https://vim.fandom.com/wiki/Modeline_magic

We can still drop it if you'd like. My personal `.vimrc` file has these settings already.